### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by d4k0** [AtomLinter/linter-csslint#20]
This deprecation message appears since Atom 0.189.0:

Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:
```json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```
```
Package.getActivationCommands (D:\Programme\Dokumente\Atom\resources\app\src\package.js:764:9)
Package.hasActivationCommands (D:\Programme\Dokumente\Atom\resources\app\src\package.js:689:20)
```

**Comment by leofle** [comment]
Same here, i updated to 0.192 and still got the same deprecation message.

**Comment by hd-deman** [comment]
fixed in 0.0.12
